src/analysis/modifications/moment.jl

```diff
@@ -0,0 +1,110 @@
+"""
+add_moment_constraints(
+    kcats::Dict{String,Float64},
+    molar_weights::Dict{String,Float64},
+    protein_mass_fraction::Float64;
+    protected_constraint_ids = ["ATPM"],
+)
+
+Add enzyme capacity constraints to problem using the MOMENT algorithm. Requires `kcats` and
+`molar_weights` for each reaction and protein. Proteins are encoded by their associated gene.
+Adds a variable vector `y` to the problem corresponding to the protein concentration. Adds a
```

Comment:
perhaps join the 2 sentences so that it's clear these are different vectors. What's the size of the vectors?